src/main/java/fact/datacorrection/DrsCalibration.java

```diff
@@ -277,15 +297,16 @@ public void init(ProcessContext processContext) throws Exception {
         }
```

Comment:
can you add the drs_url to the data item in the process method? Or a key value par containing night and run id of the drs file.

Reply:
This way it is more reproducable